src/Dev/Deprecation.php

```diff
@@ -360,6 +434,24 @@ public static function notice($atVersion, $string = '', $scope = Deprecation::SC
         }
     }
 
+    /**
+     * Shorthand method to create a suppressed notice for something with no immediate replacement.
+     * If $string is empty, then a standardised message will be used, which is:
+     * Will be removed without equivalent functionality to replace it.
```

Comment:
```suggestion
     * If $string is empty, then a standardised message will be used
```
That'll get stale if we change our mind about the message.

Reply:
Updated